app/views/books/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= @book.title %>
+<% @book.reviews.each do | review |  %>
+<%= review.body %>
+<%= review.rating %>
+<% end  %>
```

Comment:
Watch your indenting, even in a view an each loop should have its contents indented.